### PR TITLE
Pairwise preference model dev

### PR DESCRIPTION
**Pull request by WeiXiongUST**
We add a new folder to upload the pairwise preference model training code.

### CODE REVIEW

File: .DS_Store
Comment (hendrydong) [inline .DS_Store:1]: Maybe delete this one?

File: pair-pm/gemma-2b-it.yml
Comment (hendrydong) [inline pair-pm/gemma-2b-it.yml:1]: gemma-2b-it.yaml ?

File: pair-pm/llama3-8b-it.yml
Comment (hendrydong) [inline pair-pm/llama3-8b-it.yml:1]: .yml or .yaml

**Review by hendrydong — APPROVED** [review]
Some potential minor changes have been found?

File: pair-pm/llama3-8b-it.yml
Comment (WeiXiongUST) [reply pair-pm/llama3-8b-it.yml:1]: done

**Pull request merged**